tdm

```diff
@@ -167,7 +167,7 @@ fi
 tdm_curses(){
     tempfile="/tmp/tdm_$$"
     trap 'rm -f ${tempfile}' 0 1 2 3 6 14 15
-    dialog "${DEFOPT}" --menu "TDM ${VERSION}" 0 0 0 "${prglist[@]}" 2>${tempfile}
+    dialog ${DEFOPT} --menu "TDM ${VERSION}" 0 0 0 "${prglist[@]}" 2>${tempfile}
```

Comment:
I'd prefer to keep this quoted and use an array instead. Could you have a look at it ?

I think [this line](https://github.com/dopsi/console-tdm/blob/master/tdm#L140) should be the only change required